README.md

```diff
@@ -27,8 +27,16 @@ After replacing these values, run:
 ```sh
 NETWORK='rinkeby'
 GAS_PRICE_WEI='1000000000'
+INFURA_KEY="Your infura key"
 yarn deploy --network $NETWORK --gasprice $GAS_PRICE_WEI
 ```
 
 New files containing details of this deployment will be created in the `deployment` folder.
 These files should be committed to this repository.
+
+In order to verify the contract in Etherscan, run:
+
+```
+ETHERSCAN_API_KEY="your key"
+npx hardhat verify --network $NETWORK DEPLOYED_CONTRACT_ADDRESS "Constructor argument 1"
```

Comment:
Kind of a bummer that it isn't able to extract constructor arguments from the deployment file (as e.g. truffle can). Still cool though.

Reply:
Maybe we can use https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html#complex-arguments and have an `arguments.ts` that programmatically gets the constructor argument (which should be possible because of deterministic deployment :tada:).

Reply:
Don't we need to export the etherscan API key here?